EventHubs/src/index.js

```diff
@@ -6,7 +6,7 @@ var sumoHttp = require('./sumoclient');
 var dataTransformer = require('./datatransformer');
 var sumoClient;
 
-function setSourceCategory(msg, context) {
+function setSourceCategory(context, msg) {
```

Comment:
Is this really used now?

Reply:
yes 2-3 customers I know who are using it with whom I got on a call. one of the SEs recently pointed out this bug who was working with some customer.

Reply:
Ok, so this is just a place holder OOTB so yes we should fix.